pkgs/development/libraries/libopenblt/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchgit, cmake, libusb1 }:
+
+stdenv.mkDerivation rec {
+
+  pname = "libopenblt";
+  version = "011500";
+
+  src = fetchgit {
+    url = "https://github.com/feaser/openblt.git";
+    rev = "openblt_v${version}";
+    sparseCheckout = [ "Host/Source/" ];
+    sha256 = "sha256-cWdc6Imd6IBqYQRNEafp3bDDQLgdpE90fxSkY+vzhAM=";
+  };
+
+  preConfigure = "cd ./Host/Source/LibOpenBLT/";
+
+  installPhase = ''
+    cd $NIX_BUILD_TOP/openblt/Host/
+    install -D libopenblt.so $out/lib/libopenblt.so
+    cd $NIX_BUILD_TOP/openblt/Host/Source/LibOpenBLT/
+    install -D openblt.h $out/include/openblt.h
+  '';
+
+  nativeBuildInputs = [ cmake libusb1 ];
+
+  meta = with lib; {
+    description = "The OpenBLT Host Library contains an API for communicating with a microcontroller running the OpenBLT bootloader";
```

```suggestion
    description = "API for communicating with a microcontrollers running the OpenBLT bootloader";
```
See contributing doc, we don't repeat the package/library name